### ISSUE TITLE
They never update. 

### ISSUE THREAD

**Issue by thiagolucio** [dracula/atom#11]
Hi. 
The warning to "Update" never go away. I did the update process a lot of times and restart but the warning to update (at the atom footer) never get out. 

Is from your new version - 2.0.3 at the Atom 1.16.0 on Linux (ubuntu Gnome 16.4LTS and Windows 10 Last  Build)

**Comment by JimmyMultani** [comment]
I'm not sure why that would be happening. The package was renamed to `dracula-syntax` to fit better with `dracula-ui`, but that didn't cause any update issues on my Mac or Windows installations.

I would strongly suggest reinstalling the syntax theme. Sorry for the inconvenience.

**Comment by jbutkowski** [comment]
I've experienced the same issue.  It's because the package has been renamed and installing the update creates a new package folder - essentially the old package still remains installed.

You can fix it by deleting the old package 

`rm -rf ~/.atom/packages/dracula-theme`

Then reselecting the theme in atom.

**Comment by alexrussell** [comment]
I did it by uninstalling the old one just using the normal settings pane.

In fact that's a bit of a lie - I uninstalled the old one and then noticed that it said I had the new one installed (though it was not selected) so I uninstalled that too, and then I (re)installed the new one and all is good.

Lesson learnt, anyway - don't rename a package if you want a seamless update experience!

Probably worth adding this to the readme or something so people know what to do.
